Traceback (most recent call last):
  File "<string>", line 1, in <module>
    import dora; dora.start_runtime() # plot
                 ~~~~~~~~~~~~~~~~~~^^
RuntimeError: Dora Runtime raised an error.

Caused by:
   0: main task failed
   1: operator plot/op raised an error
   2: error in Python module at /Users/dave/dev/ultra/dora_webcam/plot.py
   3: Traceback (most recent call last):
        File "/Users/dave/dev/ultra/dora_webcam/plot.py", line 110, in on_event
[
          buf = pa.py_buffer(value)
  255,
        File "pyarrow/io.pxi", line 2093, in pyarrow.lib.py_buffer
  255,
        File "pyarrow/error.pxi", line 155, in pyarrow.lib.pyarrow_internal_check_status
  255,
        File "pyarrow/error.pxi", line 89, in pyarrow.lib.check_status
  255,

  56,
      TypeError: a bytes-like object is required, not 'pyarrow.lib.UInt8Array'
  2,

  0,
Location:
  0,
    binaries/runtime/src/operator/python.rs:29:9
  16,
  0,
  ...
  0,
  0,
  255,
  255,
  255,
  255,
  0,
  0,
  0,
  0
]
